client/index.html

```diff
@@ -36,11 +37,11 @@
 			</div>
 		</aside>
 		<footer id="footer">
-			<span class="tooltipped tooltipped-n tooltipped-no-touch" aria-label="Sign in"><button class="icon sign-in" data-target="#sign-in" aria-label="Sign in"></button></span>
-			<span class="tooltipped tooltipped-n tooltipped-no-touch" aria-label="Connect to network"><button class="icon connect" data-target="#connect" aria-label="Connect to network"></button></span>
-			<span class="tooltipped tooltipped-n tooltipped-no-touch" aria-label="Client settings"><button class="icon settings" data-target="#settings" aria-label="Client settings"></button></span>
-			<span class="tooltipped tooltipped-n tooltipped-no-touch" aria-label="Help"><button class="icon help" data-target="#help" aria-label="Help"></button></span>
-			<span class="tooltipped tooltipped-n tooltipped-no-touch" aria-label="Sign out"><button class="icon sign-out" id="sign-out" aria-label="Sign out"></button></span>
+			<span class="tooltipped tooltipped-n tooltipped-no-touch tooltipped-no-delay" aria-label="Sign in"><button class="icon sign-in" data-target="#sign-in" aria-label="Sign in"></button></span>
+			<span class="tooltipped tooltipped-n tooltipped-no-touch tooltipped-no-delay" aria-label="Connect to network"><button class="icon connect" data-target="#connect" aria-label="Connect to network"></button></span>
+			<span class="tooltipped tooltipped-n tooltipped-no-touch tooltipped-no-delay" aria-label="Client settings"><button class="icon settings" data-target="#settings" aria-label="Client settings"></button></span>
+			<span class="tooltipped tooltipped-n tooltipped-no-touch tooltipped-no-delay" aria-label="Help"><button class="icon help" data-target="#help" aria-label="Help"></button></span>
+			<span class="tooltipped tooltipped-n tooltipped-no-touch tooltipped-no-delay" aria-label="Sign out"><button class="icon sign-out" id="sign-out" aria-label="Sign out"></button></span>
```

Comment:
Why did you re-add `tooltipped-no-delay` to all of those? FYI, we used to not have delays on those and added them because:

- It made for a very schizophrenic UI when constantly hovering tooltipped stuff
- On mobile clicking somewhere on the menu would instantly display a tooltip + close the menu, that was very buggy

We went back and forth before we reached the current okay solution, I think we should keep it that way. That is, unless I'm missing something obvious :)

Reply:
I think the delay was increased in primer, they feel very sluggish without. Can remove

Reply:
Yeah I should have added: I am pretty sure we customized that value in our CSS so we should overwrite it in `style.css`.

In general, we made a few changes between our version and the upstream project. I am building you a diff so we can have an idea of what we're talking about.